modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -242,6 +245,22 @@ export default class LargeVideoManager {
                 const tracks = state['features/base/tracks'];
                 const videoTrack = getVideoTrackByParticipant(tracks, participant);
 
+                // Remove track streaming status listener from the old track and add it to the new track,
+                // in order to stop updating track streaming status for the old track and start it for the new track.
+                if (this.jitsiTrack) {
```

Comment:
can you document `this.jitsiTrack` in the constructor like other fields are?

Reply:
added